exchanges/graphcache/src/types.ts

```diff
@@ -183,3 +183,8 @@ export interface StorageAdapter {
 }
 
 export type Dependencies = Record<string, true>;
+
+export enum OperationType {
```

Comment:
Let's make this a const enum so it's not reflected in our outputted bundle

```suggestion
export const enum OperationType {
```